tests/test_connection.py

```diff
@@ -420,3 +424,138 @@ def test_no_mfa_passcode(mock_connect, runner):
     assert result.exit_code == 0, result.output
     args, kwargs = mock_connect.call_args
     assert kwargs.get("passcode") == None
+
+
+@pytest.mark.parametrize(
+    "env",
+    [
+        {
+            "SNOWFLAKE_CONNECTIONS_EMPTY_ACCOUNT": "some_account",
+            "SNOWFLAKE_CONNECTIONS_EMPTY_DATABASE": "test_database",
+            "SNOWFLAKE_CONNECTIONS_EMPTY_WAREHOUSE": "large",
+            "SNOWFLAKE_CONNECTIONS_EMPTY_ROLE": "role",
+            "SNOWFLAKE_CONNECTIONS_EMPTY_SCHEMA": "my_schema",
+            "SNOWFLAKE_CONNECTIONS_EMPTY_PASSWORD": "dummy",
+        },
+        {
+            "SNOWFLAKE_ACCOUNT": "some_account",
+            "SNOWFLAKE_DATABASE": "test_database",
+            "SNOWFLAKE_WAREHOUSE": "large",
+            "SNOWFLAKE_ROLE": "role",
+            "SNOWFLAKE_SCHEMA": "my_schema",
+            "SNOWFLAKE_PASSWORD": "dummy",
+        },
+    ],
+)
+@mock.patch("snowflake.connector.connect")
+def test_connection_details_are_resolved_using_also_generic_variables(
```

Comment:
generic variables values are the same as "empty" connection values, so IMO the test isn't confirming what its name  is suggesting

Reply:
Adjusted test name